src/test/fuzz/deserialize.cpp

```diff
@@ -183,7 +184,10 @@ void test_one_input(const std::vector<uint8_t>& buffer)
 #elif NETADDR_DESERIALIZE
         CNetAddr na;
         DeserializeFromFuzzingInput(buffer, na);
```

Comment:
The test would be more powerful if it permitted constructing the input using ADDRV2_FORMAT (or have the option for both), as no torv3/i2p/cjdns addresses will be construction currently.

Reply:
Note that the current version permits constructing the input using `ADDRV2_FORMAT` since the version is read from the input provided by the fuzzer:

https://github.com/bitcoin/bitcoin/blob/903f3d06275312aa4000b765d2287339210c61c1/src/test/fuzz/deserialize.cpp#L64-L70

In other words torv3/i2p/cjdns addresses can be construction currently :)

Reply:
Ooh, very good. Ignore me.